### PR TITLE
FWF-3701 : BPM upgrade to latest version

### PR DESCRIPTION
**Pull request by sumesh-aot**
# Issue Tracking

JIRA: 
Issue Type: FWF-3701

# Changes
- Upgraded spring boot from 3.1 to 3.3
- Upgraded camunda from 7.20 to 7.21
- Upgraded spring-websocket to 6.1

# Screenshots (if applicable)
Screenshot 2024-12-12 at 3 33 00 PM

# Notes

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

File: forms-flow-bpm/forms-flow-bpm-camunda/src/main/resources/META-INF/resources/webjars/camunda/app/cockpit/styles/user-styles.css [hunk 1]
```diff
@@ -74,4 +74,8 @@ aside li.active>a {
   font-size: 30px;
   padding: 7px;
   line-height: 36px;
-}
\ No newline at end of file
+}
+
+.ce-eol-banner {
```
Comment (sumesh-aot) [inline forms-flow-bpm/forms-flow-bpm-camunda/src/main/resources/META-INF/resources/webjars/camunda/app/cockpit/styles/user-styles.css:79]: This change is to hide the red notice from Camunda on end of life of v7 by Oct 2025

**Review by arun-s-aot — APPROVED** [review]

File: forms-flow-bpm/forms-flow-bpm-camunda/pom.xml [hunk 2]
```diff
@@ -28,10 +28,10 @@
 
         <!-- versions -->
         <version.camundaKeycloak>7.21.5</version.camundaKeycloak>
-        <version.camunda>7.20.0</version.camunda><!-- 7.18.0 - 7.20.0 -->
+        <version.camunda>7.21.0</version.camunda><!-- 7.18.0 - 7.20.0 -->
```
Comment (auslin-aot) [inline forms-flow-bpm/forms-flow-bpm-camunda/pom.xml:31]: comments on the right indicate the old version. will you please update that too.

File: forms-flow-bpm/forms-flow-bpm-camunda/pom.xml [hunk 3]
```diff
@@ -28,10 +28,10 @@
...
         <version.camundaConnect>1.5.4</version.camundaConnect><!-- 1.5.4 -->
         <version.camundaMail>1.5.0</version.camundaMail><!-- 1.5.0 -->
-        <version.springBoot>3.1.10</version.springBoot><!-- 3.1.5 - 3.1.10 -->
+        <version.springBoot>3.3.5</version.springBoot><!-- 3.1.5 - 3.1.10 -->
```
Comment (auslin-aot) [inline forms-flow-bpm/forms-flow-bpm-camunda/pom.xml:34]: here too

File: forms-flow-bpm/pom.xml [hunk 4]
```diff
@@ -23,10 +23,10 @@
 
 		<!-- versions -->
 		<version.camundaKeycloak>7.21.5</version.camundaKeycloak>
-		<version.camunda>7.20.0</version.camunda><!-- 7.18.0 - 7.20.0 -->
+		<version.camunda>7.21.0</version.camunda><!-- 7.18.0 - 7.20.0 -->
 		<version.camundaConnect>1.5.4</version.camundaConnect><!-- 1.5.4 -->
 		<version.camundaMail>1.5.0</version.camundaMail><!-- 1.5.0 -->
-		<version.springBoot>3.1.10</version.springBoot><!-- 3.1.5 - 3.1.10 -->
+		<version.springBoot>3.3.5</version.springBoot><!-- 3.1.5 - 3.1.10 -->
```
Comment (auslin-aot) [inline forms-flow-bpm/pom.xml:29]: here too

File: forms-flow-bpm/forms-flow-bpm-camunda/pom.xml (hunk 2 above)
Comment (sumesh-aot) [reply forms-flow-bpm/forms-flow-bpm-camunda/pom.xml]: Done

File: forms-flow-bpm/forms-flow-bpm-camunda/pom.xml (hunk 3 above)
Comment (sumesh-aot) [reply forms-flow-bpm/forms-flow-bpm-camunda/pom.xml]: Done

File: forms-flow-bpm/pom.xml (hunk 4 above)
Comment (sumesh-aot) [reply forms-flow-bpm/pom.xml]: Done

**Review by auslin-aot — APPROVED** [review]

**Pull request merged**